bootstrap/utils/utils.go

```diff
@@ -161,3 +167,47 @@ func DeepCopy(src any, dest any) error {
 	}
 	return nil
 }
+
+func LoadFile(path string, contents *any) error {
+	var fileBytes []byte
+	var err error
+
+	lowerPath := strings.ToLower(path)
+
+	if strings.Contains(lowerPath, "http") {
```

Comment:
This needs to be a `Starts with` 
```suggestion
	if strings.Index(lowerPath, "http") == 0 {
```